tasks/install.deb.yml

```diff
@@ -2,6 +2,22 @@
 
 - include_vars: "{{ansible_distribution}}.yml"
 
```

Comment:
I think all this code should be after mongo package installation. And I don't know why you use flush_handlers? After installation and before auth you need to ensure that service is started. If you flushed hadlers after configuration and config have replica_set - you will not able to configure auth, because mongo will say you 'I am not a master, go home dude :)'.